docs/02-for-app-authors/03-metainfo-guidelines/01-quality-guidelines.md

```diff
@@ -26,9 +26,9 @@ If you're not sure if your app violates any trademarks, feel free to ask us on [
 
 Use Flathub verification to indicate that you are the author of the app and that you are managing the app on Flathub.
 
-### Brand color
+### Brand colors provided
```

Comment:
I'm not sure, seems implicit that this is about the color being provided no? I'd maybe instead focus on the quality of the brand colors, given that we'd not just accept any brand colors but would reject e.g. a very light color as the dark variant. Maybe it could be "Good brand colors"?

Or actually, maybe we should have two checks for this:
- An automatic one that just checks if there's two colors provided called "Brand colors provided"
- A manual one called "Good brand colors"

Reply:
I guess we need that, but I'm not keen to have more checks :D

The internal name for the guideline is `has-primary-branding` right now for context

Reply:
Perhaps we could have a single manually toggleable check, but the system would automatically pre-set it to false (and make it non-toggleable) if there's <2 colors provided?

Reply:
We agreed to split this up into two guidlines